### PR TITLE
Lh OKTA-522912 text update on SIW Challenge Screen

### PR DESCRIPTION
**Pull request by lokeshhemrajani-okta**
## SIW Challenge Screen text update
- Text updated on Challenge screen for 'Enter code' link

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-522912](https://oktainc.atlassian.net/browse/OKTA-522912)

### Reviewers:
@okta/ciamx 

### Screenshot/Video:
Screen Shot 2022-08-12 at 11 44 41 AM
Screen Shot 2022-08-12 at 11 39 20 AM

### Downstream Monolith Build:
NA

### CODE REVIEW

**Review by nikhilvenkatraman-okta — APPROVED** [review]

File: test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js [hunk 1]
```diff
@@ -320,8 +320,10 @@ test
     await t.expect(saveBtnText).eql('Verify');
 
     const emailAddress = emailVerification.currentAuthenticatorEnrollment.value.profile.email;
+    const enterVerificationCodeText = challengeEmailPageObject.getEnterVerificationCodeText();
     await t.expect(challengeEmailPageObject.getFormSubtitle())
       .eql(`We sent an email to ${emailAddress}. Click the verification link in your email to continue or enter the code below.`);
+    await t.expect(enterVerificationCodeText).eql('Enter a verification code instead');
```
Comment (justinabrokwah-okta) [inline test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js:326]: nit: Make the string `'Enter a verification code instead'` a constant so you can reuse it within tests

**Review by justinabrokwah-okta — APPROVED** [review]
LGTM!

**Review by vicentevillegas-okta — APPROVED** [review]
LGTM

👍

File: test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js (hunk 1 above)
Comment (lokeshhemrajani-okta) [reply test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js]: done

**Review by zihanwang-okta — APPROVED** [review]

**Review by mauriciocastillosilva-okta — APPROVED** [review]
Please follow up the week of monthly release to be sure the new string has translations to all languages

**Review by shuowu-okta — APPROVED** [review]

**Pull request merged**